HACKING.md

```diff
@@ -0,0 +1,17 @@
+This is a plugin for cockpit.
+Install cockpit according to https://cockpit-project.org/running.html
+
+Pull the repository and compile
+
+```sh
+$ git clone https://github.com/skobyda/cockpit-certificates.git
+$ cd cockpit-certificates
+$ make
+```
+
+This project is based on cockpit's [starter-kit](https://github.com/cockpit-project/starter-kit)
```

Comment:
Please put a period after this, otherwise it [is hard to read](https://github.com/skobyda/cockpit-certificates/blob/cc0312769e5178aab5084a19470283eb081e90f1/HACKING.md). Same goes for the first paragraph.